docker-hub/billing/faq.md

```diff
@@ -1,211 +1,83 @@
-
-### Are there rate limits on Docker Hub container image requests?
-
-Rate limits for unauthenticated and free Docker Hub usage went into effect on November 1, 2020. More information about this is available in [this blog post](https://www.docker.com/blog/docker-hub-image-retention-policy-delayed-and-subscription-updates/){: target="blank" rel="noopener" class=""}.
-
-### How is a pull request defined for purposes of rate limiting?
-
-A pull request is up to two GET requests to the registry URL path `/v2/*/manifests/*`.
-
-This accounts for the fact that container pull requests for multi-arch images require a manifest list to be downloaded followed by the actual image manifest for the required architecture. HEAD requests are not counted.
-
-Note that all pull requests, including ones for images you already have, are counted by this method. This is the trade-off for not counting individual layers.
-
-### Can I use a Docker Pro subscription to run a service account that makes unlimited container pull requests to Docker Hub on behalf of an organization?
-
-Docker Pro subscriptions are designed for use by a single individual. Daily pulls are limited to 50,000 image pulls from Docker Hub per 24 hours.
-
-### Can I run a local mirror of Docker Hub?
-
-Please see [https://docs.docker.com/registry/recipes/mirror/](https://docs.docker.com/registry/recipes/mirror/) to run [docker/distribution](https://github.com/docker/distribution){: target="blank" rel="noopener" class=""} as a mirror. Because distribution uses HEAD manifest requests to check if cached content has changed, it will not count towards the rate limit. Note that initial requests for Hub images that are not yet in cache will be counted.
+### Do collaborator limits differ between the Free and Pro plans?
 
-### Are image layers counted?
+The Free plan includes unlimited collaborators for public repositories and 0 collaborators for private repositories. Pro includes unlimited collaborators for public repositories and 1 unique collaborator for private repositories. Note that a collaborator on a private repository within a Pro plan can only be used with a service account. For details on the differences between plans, see (https://www.docker.com/pricing).
 
-No. Because we are limiting on manifest requests, the number of layers (blob requests) related to a pull is unlimited at this time. This is a change from our previous policy based on community feedback in order to be more user-friendly, so users do not need to count layers on each image they may be using.
+### Where can I renew my Docker Enterprise subscription??
 
-### Are anonymous (unauthenticated) pulls rate-limited based on IP address?
+The former Docker Enterprise product is now Mirantis Enterprise Platform. You can contact Mirantis for more information by visiting their [website](https://www.mirantis.com/%20target=).
```

Comment:
Is this URL valid? Seems like the "%20target=" should be something like [website](https://www.mirantis.com){:target="_blank"}

Reply:
Updated with new URL